samples/subsys/ipc/ipc_service/icmsg/remote/boards/stm32h747i_disco.overlay

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2024 Celina Sophie Kalus <hello@celinakalus.de>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	chosen {
+		/delete-property/ zephyr,ipc_shm;
+	};
+
+	/* HSEM IPM is incompatible with HSEM MBOX */
+	/delete-node/ mailbox;
+
+	/* Define new memory regions for TX and RX */
+	/delete-node/ memory@38000000;
+
+	sram_rx: memory@38000000 {
+		reg = <0x38000000 0x08000>;
+	};
+
+	sram_tx: memory@38008000 {
+		reg = <0x38008000 0x08000>;
+	};
+};
+
+&pinctrl {
+	mbox: mbox@58026400 {
```

Comment:
I was cautious of doing that because of the already existing IPM mailbox which is already used and is in conflict with this MBOX (if one uses the same HSEMs). I would be thankful for any hints how to proceed in a manner that preserves compatibility with existing code.

Reply:
These DT nodes have now been merged together and the MBOX is defined in the device tree of the board instead of the overlay.